package.json

```diff
@@ -64,9 +64,9 @@
 		"@types/validator": "^13.15.1",
 		"copyfiles": "^2.4.1",
 		"eslint": "^9.28.0",
+		"eslint-config-love": "^120.0.0",
 		"eslint-config-prettier": "^10.1.5",
 		"eslint-import-resolver-typescript": "^4.4.3",
-		"eslint-plugin-import": "^2.31.0",
```

Comment:
Just calling out: love includes plugin import as a dependency so we no longer need to!  This is, I think, one of the points of the flat config approach / we don't need to set up peer dependencies of configuration sets manually.